src/test/resources/template/ParameterReuseRecipe$1_before.java

```diff
@@ -16,7 +16,19 @@
 package foo;
 import org.openrewrite.java.*;
 
+/**
+ * OpenRewrite `before` template created for  `foo.ParameterReuseRecipe$1`.
+ */
 public class ParameterReuseRecipe$1_before {
+    /**
+     * Instantiates a new {@link ParameterReuseRecipe$1_before} instance.
+     * @return the new instance
+     */
+    public ParameterReuseRecipe$1_before() {}
+
+    /**
+     * @return `JavaTemplate` to match or replace.
```

Comment:
It _might_ be that doclint will complain here about the absence of a summary line, but not sure.

Reply:
Indeed it did; fixed with an extra line.